.github/workflows/release.yml

```diff
@@ -105,4 +105,458 @@ jobs:
+      - name: Sign Published Docker image using Cosign with annotation from metadata
+        run: |
+          cosign sign --key cosign.key \
+            -a "tags"="${{ steps.meta.outputs.tags }}" \
+            -a "labels=${{ steps.meta.outputs.labels }}" \
+            -a "owner"=${{ github.repository_owner }}" \
+            -a "repo=${{ github.repository }}" \
+            -a "actor=${{ github.actor }}" \
+            -a "trigerringactor=${{ github.triggering_actor}}" \
+            -a "ref=${{ github.ref }}" \
+            -a "sha=${{ github.sha }}" \
+            -a "workflowref=${{ github.workflow_ref }}" \
+            -a "workflowsha=${{ github.workflow_sha }}" \
+            -a "type=paralus" \ 
+            ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ steps.meta.outputs.tags }}
+
+          cosign sign --key cosign.key \
+            -a "tags"="${{ steps.meta-init.outputs.tags }}" \
+            -a "labels=${{ steps.meta-init.outputs.labels }}" \
+            -a "owner"=${{ github.repository_owner }}" \
+            -a "repo=${{ github.repository }}" \
+            -a "actor=${{ github.actor }}" \
+            -a "trigerringactor=${{ github.triggering_actor}}" \
+            -a "ref=${{ github.ref }}" \
+            -a "sha=${{ github.sha }}" \
+            -a "workflowref=${{ github.workflow_ref }}" \
+            -a "workflowsha=${{ github.workflow_sha }}" \
+            -a "type=paralus-init" \ 
+            ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}-init:${{ steps.meta-init.outputs.tags }}
+
+          cosign sign --key cosign.key \
+            -a "tags"="${{ steps.meta-sync.outputs.tags }}" \
+            -a "labels=${{ steps.meta-sync.outputs.labels }}" \
+            -a "owner"=${{ github.repository_owner }}" \
+            -a "repo=${{ github.repository }}" \
+            -a "actor=${{ github.actor }}" \
+            -a "trigerringactor=${{ github.triggering_actor}}" \
+            -a "ref=${{ github.ref }}" \
+            -a "sha=${{ github.sha }}" \
+            -a "workflowref=${{ github.workflow_ref }}" \
+            -a "workflowsha=${{ github.workflow_sha }}" \
+            -a "type=kratos-synchronizer" \ 
+            ${{ env.REGISTRY }}/paralusio/kratos-synchronizer:${{ steps.meta-sync.outputs.tags }}
+
+      # Removing private key from the local runner
+      # Sensitive information from local environments can be leaked.
+      # Using Key Management Services is recommended.
+      - name: Clean up the Cosign private key
+        run: |
+          rm cosign.key
+
+      - name: Write public key to disk
+        shell: bash
+        run: |
+            'echo "$KEY" > cosign.pub'
+        env:
+          KEY: ${{ secrets.COSIGN_PUBLIC_KEY }}
+      
+      # Using external action for simplicity sake.
+      # There codebase is available for public on GitHub thus vulnerabilities can be discovered if any.
+      - name: The Docker CLI
```

Comment:
Do we really need to install Docker CLI separately? I believe it should be available with docker actions we have used above.
I will confirm this and get back.

Reply:
Yeah, this one I wasn't confirmed about. Checked it but I guess it isn't with the action that has been used above so added it. But if it is there let me know I will remove it.